.github/workflows/release.yml

```diff
@@ -34,16 +40,33 @@ jobs:
   deploy-docker:
     runs-on: ubuntu-latest
 
+    if: ${{ always() }}
     needs: deploy-pypi
     steps:
       - uses: actions/checkout@v2
-      - name: Login
-        env:
-          DOCKER_USER: ${{secrets.DEVOPSHQ_DOCKER_USER}}
-          DOCKER_PASSWORD: ${{secrets.DEVOPSHQ_DOCKER_TOKEN}}
-        run: |
-          docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
-      - name: Build
-        run: docker build . --file docker/Dockerfile --tag devopshq/artifactory-cleanup:latest
-      - name: Push
-        run: docker push devopshq/artifactory-cleanup:latest
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: devopshq/artifactory-cleanup
+          flavor: latest=true
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+
+      - name: Login to image repository
```

Comment:
This one fails with `Error: Username and password required` 
I'm not sure why it happens- looks like actions have all access and were run from artifactory-cleanup, not a fork repository